lib/service/Service.ts

```diff
@@ -96,11 +98,27 @@ class Service {
     argChecks.VALID_SWAP_CLIENT(args);
     const { currency, swapClient, tokenAddress, decimalPlaces } = args;
 
+    let address = tokenAddress;
+    if (args.swapClient === SwapClientType.Raiden && !!address) {
+      address = address.toLowerCase().replace('0x', '');
```

Comment:
Could we move the conversion logic to to `utils.ts` and use `toChecksumAddress` in this function?

Reply:
Sure, seems like a good idea. Will do so.